brooklyn-server/core/src/test/java/org/apache/brooklyn/core/catalog/internal/CatalogDtoTest.java

```diff
@@ -111,6 +113,9 @@ public CatalogDto buildExampleCatalog() {
         testEntitiesJavaCatalog.addEntry(CatalogItemBuilder.newTemplate(TestApplication.class.getCanonicalName(), BasicBrooklynCatalog.NO_VERSION)
                 .displayName("Test App from JAR")
                 .javaType(TestApplication.class.getCanonicalName())
+                .tag(BrooklynTags.newNotesTag("Some notes for catalog testing"))
```

Comment:
Should we test in this method that we can get these tags back out again?
And explicitly test rebind, to confirm the tags are still there?